### ISSUE TITLE
Issues with generation for C and C++ with 0.12.0

### ISSUE THREAD

**Issue by dpuleri** [nikhilkalige/docblockr#291]
For a function that looks like the following:

 ```cpp
int someFunction(int a, int b, std::vector<int> vec) {
    return -1;
}
```

the documentation used to look something like this (with `@param` and `@return` showing):

```cpp
/**
 * [someFunction description]
 * @param a      [description]
 * @param b      [description]
 * @param vec  [description]
 */
int someFunction(int a, int b, std::vector<int> vec) {
    return -1;
}
```

but now it looks like this:

```cpp
/**
 * [std description]
 * @type {[type]}
 */
int someFunction(int a, int b, std::vector<int> vec) {
    return -1;
}
```

This change in the generated documentation happened after the update to 0.12.0.

**Comment by MoritzKn** [comment]
Yeah, I can confirm this to be a regression with `v0.12.0`.

**Comment by MoritzKn** [comment]
The bug was introduced with #290, fixed by 997b6e4. WIth `v0.12.1` everything should work fine again.
Thank for reporting!

**Comment by dpuleri** [comment]
Wow awesome! Thanks for fixing that so quickly.
